Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM node:6.5
+
+WORKDIR /app
+COPY package.json /app/
+RUN npm install --loglevel warn
+RUN npm config set unsafe-perm true
+COPY . /app/
+
+CMD ["npm", "test"]
```

Comment:
I put the docker test command into the `package.json`, so just run `npm run docker-test`, it will run the test in Docker.